applications/GeoMechanicsApplication/tests/hexa_8n_normal_load/MaterialParameters.json

```diff
@@ -0,0 +1,29 @@
+{
+   "properties": [{
+      "model_part_name":         "PorousDomain.Soil",
+      "properties_id":           1,
+      "Material": {
+          "constitutive_law": {
+              "name"             :  "LinearElastic3DLaw" 
+          },
+          "Variables": {
+              "IGNORE_UNDRAINED"         :  true,
+              "YOUNG_MODULUS"            :  3.0e7,
+              "POISSON_RATIO"            :  0,
+              "DENSITY_SOLID"            :  0,
+              "DENSITY_WATER"            :  1.0e3,
```

Comment:
Just a question: is there any particular reason why we set the density of the solid matter to zero, but the density of water to its actual value (assuming SI units)?

Reply:
No clue, copied from another test. It is not used in this test, since it's static (no mass matrix involved anyway), so when giving it a value, it still passes. Should we add the density in that case?

FYI @WPK4FEM

Reply:
I would suggest to discuss this with the team to come up with a consistent strategy. We can then apply it consistently to all of our tests. So, no action required for this PR.